frontends/web/src/components/forms/checkbox.module.css

```diff
@@ -69,19 +69,19 @@
 .success input:checked + label::before {
     background-color: var(--color-success);
     border-color: var(--color-success);
-    color: var(--color-white);
+    color: var(--color-alt);
 }
 
 .warning input:checked + label::before {
     background-color: var(--color-warning);
     border-color: var(--color-warning);
-    color: var(--color-white);
+    color: var(--color-alt);
 }
 
 .info input:checked + label::before {
     background-color: var(--color-info);
     border-color: var(--color-info);
-    color: var(--color-white);
+    color: var(--color-alt);
 }
 
```

Comment:
There is more in this file, see line 14 `background: rgb(255, 255, 255);` can be changed to `background: var(--background-secondary);`

Reply:
changed